src/components/Pagination/Pagination.js

```diff
@@ -81,17 +81,20 @@ export const pagination = WrappedComponent =>
               />
             </Col>
             <Col className="text-right">
-              <select
-                value={this.state.page}
-                onChange={this.handlePageChange}
-                onBlur={this.handlePageChange}
-              >
-                {this.pagerCounter().map((value, i) => (
-                  <option value={value} key={i}>
-                    Page {value}
-                  </option>
-                ))}
-              </select>
+              <div className="select">

```

Comment:
I actually have a PR out to replace most of this implementation, would you mind reverting these changes?

Reply:
Done. Hope to reuse the select element at some point though.